src/fprime/fbuild/cli.py

```diff
@@ -118,6 +83,47 @@ def run_fbuild_cli(
         )
 
 
+def purge(build: Build, parsed: argparse.Namespace):
+    for purge_build in Build.get_build_list(
+        build, parsed.build_cache, ignore_invalid=parsed.force
+    ):
+        print(
+            f"[INFO] {parsed.command.title()} build directory at: {purge_build.build_dir}"
```

Comment:
This causes some unexpected output
```
[INFO] Generate build directory at: fprime/Ref/build-fprime-automatic-native
[INFO] Generate build directory at: fprime/Ref/build-fprime-automatic-native-ut
[INFO] Skipping purge. The following directory does not exist: fprime/Ref/build-fprime-automatic-native-ut
[INFO] Generating build directory at: fprime/Ref/build-fprime-automatic-native
[INFO] Using toolchain file None for platform default
-- The C compiler identification is AppleClang 16.0.0.16000026
```

Since we're purging with the `parsed.command` = `generate`
The easiest is probably to hardcode the string, e.g. `"[INFO] Purging build dir...."

Reply:
(This would not need fixing if we were to go with the above suggestion)